pyproject.toml

```diff
@@ -286,15 +284,12 @@ max-complexity = 21
 "nilearn/glm/tests/test_thresholding.py" = ["D103"]
 "nilearn/glm/tests/test_utils.py" = ["D103"]
 "nilearn/image/**/{tests}/*" = ["D103"]  # 75 errors to fix
-"nilearn/maskers/*" = ["B028"]  # 17 errors to fix
-"nilearn/mass_univariate/*" = ["B028"]  # 4 errors to fix
 "nilearn/plotting/*" = ["B028"]  # 18 errors to fix
```

Comment:
@hndgzkn 
will not touch plotting module for now because it is a 'moving target' while refactoring